platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -253,7 +270,7 @@ - (void)pickFeatureAt:(CGPoint)screenPosition
     screenPosition.x *= self.contentScaleFactor;
     screenPosition.y *= self.contentScaleFactor;
 
-    self.map->pickFeatureAt(screenPosition.x, screenPosition.y, [screenPosition, self](const Tangram::FeaturePickResult* featureResult) {
+    self.map->pickFeatureAt(screenPosition.x, screenPosition.y, [=](const Tangram::FeaturePickResult* featureResult) {
```

Comment:
Why switching the closure to copy all? Will a reference to `self` not suffice? Or `self` is asyncly updated?